services/graylog/docker-compose.yml

```diff
@@ -53,6 +56,10 @@ services:
       # Username: admin
       - GRAYLOG_ROOT_PASSWORD_SHA2=${GRAYLOG_ROOT_PASSWORD_SHA2}
       - GRAYLOG_HTTP_EXTERNAL_URI=${GRAYLOG_HTTP_EXTERNAL_URI}
+      # Loading & install our custom content pack
+      - GRAYLOG_CONTENT_PACKS_LOADER_ENABLED=true
+      - GRAYLOG_CONTENT_PACKS_DIR=/files
+      - GRAYLOG_CONTENT_PACKS_AUTO_INSTALL=simcore-content-pack.json
```

Comment:
that is a nice addition. is this new from the latest version of graylog?

Reply:
I don't think so, I found something about them even in version 3.